src/bootstrap/native.rs

```diff
@@ -701,6 +701,15 @@ fn supported_sanitizers(
                 });
             }
         }
+        "aarch64-unknown-linux-gnu" => {
```

Comment:
At this point I think it might make some sense to figure out how to extract the loop part into a function to avoid so much code duplication. Ideally we end up with 1 line in the body of each match statement.

At that point it then would become easier to also just add support for other targets too.

Reply:
Refactored a little bit. Unfortunately the naming scheme is just different between the toolchains.